### DISCUSSION TITLE
Is there an API for the progress of the shipment (that is, the progress of the shipment sent by the seller to Amazon FBA)

### DISCUSSION THREAD

**Discussion by RaKaiYli** [amzn/selling-partner-api-models#3213] [General]
Is there an API for the progress of the shipment (that is, the progress of the shipment sent by the seller to Amazon FBA)？？？

**Comment by NetRushJacob** [comment]
That would be the Fulfillment Inbound API:
https://github.com/amzn/selling-partner-api-docs/blob/main/references/fulfillment-inbound-api/fulfillmentInboundV0.md
Specifically: GetShipments, GetShipmentItemsByShipmentId, GetShipmentItems
